community-plugins.json

```diff
@@ -381,7 +381,7 @@
         "id": "obsidian-to-anki-plugin",
         "name": "Export to Anki",
         "author": "Pseudonium",
-        "description": "This is an Anki integration plugin! Designed for efficient bulk exporting.",
+        "description": "Previously known as Obsidian_to_Anki plugin. This is an Anki integration plugin! Designed for efficient bulk exporting.",
```

Comment:
Along with the name changes, we have also fixed plugin descriptions.
The new description for your plugin is:
> Anki integration designed for efficient bulk exporting.

Please refactor your PR accordingly.